src/util/mod.rs

```diff
@@ -0,0 +1 @@
+pub mod environment;
```

Comment:
Although we don't need this with the previous comment, for future reference, you don't need a separate file here, you could use an inline module in the parent.